src/raftstore/store/store.rs

```diff
@@ -890,6 +890,7 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             let mut ctx = ReadyContext::new(&mut self.raft_metrics, &self.trans, pending_count);
             for region_id in self.pending_raft_groups.drain() {
                 if let Some(peer) = self.region_peers.get_mut(&region_id) {
+                    peer.schedule_apply_proposals(&mut self.apply_worker);
```

Comment:
we can add a local metric here to get proposals of one region in one raft ready. 
If many regions have one or two proposals, we may have a scattered write, otherwise, we will have a centralized write.